src/slider/slider.component.ts

```diff
@@ -146,6 +146,9 @@ export class Slider implements AfterViewInit, OnDestroy, ControlValueAccessor {
 
 		this._value = v;
 		this.slidAmount = this.convertToPx(v);
+		setTimeout(() => {
```

Comment:
Don't think you need the curly brackets. And maybe make it one line?

Reply:
Sure :+1: